sql/updates/world/master/2022_02_07_03_world_XX.sql

```diff
@@ -0,0 +1,31 @@
+SET @GARRICK = 1568280;
+SET @GARRICKGUID = -1050145;
+
+-- Captain Garrick (Remove SmartAI and add ScriptName)
+UPDATE creature_template SET AIName = '', ScriptName = 'npc_captain_garrick' WHERE entry = @GARRICK;
+DELETE FROM smart_scripts WHERE entryorguid = @GARRICKGUID;
```

Comment:
also check `source_type`, you are currently also deleting gameobjects, areatrigger, timedactionlists, ... with that entryorguid